packages/sdk/src/persistenceStore.ts

```diff
@@ -105,12 +106,15 @@ export interface IPersistenceStore {
         rangeStart: bigint,
         randeEnd: bigint,
     ): Promise<ParsedMiniblock[]>
+    saveSnapshot(streamId: string, miniblockNum: bigint, snapshot: Snapshot): Promise<void>
+    saveSnapshots(streamId: string, snapshots: Record<string, Snapshot>): Promise<void>
 }
 
 export class PersistenceStore extends Dexie implements IPersistenceStore {
     cleartexts!: Table<{ cleartext: Uint8Array | string; eventId: string }>
     syncedStreams!: Table<{ streamId: string; data: Uint8Array }>
     miniblocks!: Table<{ streamId: string; miniblockNum: string; data: Uint8Array }>
+    snapshots!: Table<{ streamId: string; miniblockNum: string; data: Uint8Array }>
```

Comment:
we don't want to save these snapshots - it's gigs of data that the client will never read.